pam/pam_test/module-transaction-dummy_test.go

```diff
@@ -0,0 +1,512 @@
+package pam_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/msteinert/pam"
+	"github.com/stretchr/testify/require"
+)
+
+func ptrValue[T any](value T) *T {
+	v := value
+	return &v
+}
+
+func TestSetGetItem(t *testing.T) {
+	t.Parallel()
+	t.Cleanup(MaybeDoLeakCheck)
+	t.Cleanup(MaybeDoLeakCheck)
+
+	testCases := map[string]struct {
+		item             pam.Item
+		value            *string
+		expectedGetError error
+		expectedSetError error
+		expectedValue    *string
```

Comment:
Nitpick: Can you reorder this so `expectedValue` comes first and add a blank line between what we expect as input and what we expect as output?
e.g.
```
item
value

expectedValue
...
```